workflow.cwl

```diff
@@ -422,20 +422,20 @@ steps:
   #       source: "#scoring/results"
   #   out: []
 
-  # annotate_submission_with_output:
-  #   run: https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v3.0/cwl/annotate_submission.cwl
-  #   in:
-  #     - id: submissionid
-  #       source: "#submissionId"
-  #     - id: annotation_values
-  #       source: "#scoring/results"
-  #     - id: to_public
-  #       default: true
-  #     - id: force
-  #       default: true
-  #     - id: synapse_config
-  #       source: "#synapseConfig"
-  #     - id: previous_annotation_finished
-  #       source: "#annotate_validation_with_output_real/finished"
-  #   out: [finished]
+  annotate_submission_with_output:
```

Comment:
The problem with this is that we would be overwriting the annotations from the "fake" data

Reply:
Hm, what do you mean?

I had removed the step to score the models against the synthetic data; instead, it should stealthily (lol) be scoring against the real data.

Reply:
Got it.  This was only relevant if you wanted to save the synthetic data scores.